internal/observability/graph_test.go

```diff
@@ -38,6 +38,7 @@ func TestEdgeNopFallbacks(t *testing.T) {
 	req := &transport.Request{
 		Caller:          "caller",
 		Service:         "service",
+		Transport:       unknownIfEmpty(""),
```

Comment:
This is just "unknown" - why hop through the extra function call?

Reply:
done